dockerfiles/buildall.sh

```diff
@@ -25,6 +25,7 @@ BUILD_TAG=$(git describe --exact-match --tags)  # The git tag for the images we
 BUILD_LABEL=${BUILD_TAG}                        # The docker hub label for all other components
 BUILD_USER=hpcc-systems                         # The github repo owner
 BUILD_TYPE=                                     # Set to Debug for a debug build, leave blank for default (RelWithDebInfo)
+INCLUDE_CPPUNIT=1
```

Comment:
I think this is supposed to be USE_CPPUNIT?
"INCLUDE_CPPUNIT" isn't used anywhere afaics

Reply:
Good catch. Have fixed